### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by rita-savaliya-scaletech**
## Design

- Login, Signup, Guest login  
- UUID for  guest user

### CODE REVIEW

File: src/features/auth/container/login.tsx [hunk 1]
```diff
@@ -1,13 +1,32 @@
 import React from 'react';
-
+import Lottie from 'react-lottie';
 import LoginForm from '../component/loginForm';
+import diceAnimation from 'assets/lotties/diceAnimation.json';
```
Comment (jasmeen-goyani-scaletech) [inline src/features/auth/container/login.tsx:4]: Follow import rules

File: src/features/auth/container/login.tsx [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
+	};
+
 	return (
-		<div className='flex justify-content--center align-items--center flex--column height--full-viewport'>
-			<p className='auth-title text--center text--white font-size--28'>Log In</p>
-			<div className='auth_form width--50 m--0-auto form'>
-				<LoginForm />
+		<div className='flex justify-content--around align-items--center height--full-viewport'>
+			<div className='login-wrapper border-radius--lg width--30'>
+				<div className=''>
```
Comment (jasmeen-goyani-scaletech) [inline src/features/auth/container/login.tsx]: Remove class name

**Review by jasmeen-goyani-scaletech — CHANGES_REQUESTED** [review]

File: src/features/auth/container/login.tsx (hunk 1 above)
Comment (rita-savaliya-scaletech) [reply src/features/auth/container/login.tsx:4]: Done

File: src/features/auth/container/login.tsx (hunk 2 above)
Comment (rita-savaliya-scaletech) [reply src/features/auth/container/login.tsx]: Done

**Review by jasmeen-goyani-scaletech — APPROVED** [review]

**Pull request merged**